backend/api/Controllers/RobotController.cs

```diff
@@ -822,5 +822,56 @@ await missionSchedulingService.AbortAllScheduledNormalMissions(
 
             return NoContent();
         }
+
+        /// <summary>
+        ///     Resume paused mission on a robot
+        /// </summary>
+        /// <remarks>
+        ///     <para> This query resumes the currently paused mission for a robot </para>
+        /// </remarks>
+        [HttpPost]
+        [Authorize(Roles = Role.Admin)]
```

Comment:
This is probably fine for now, but in the future we might want to allow users to do it too.